h/auth/__init__.py

```diff
@@ -41,6 +44,15 @@ def includeme(config):
     global DEFAULT_POLICY
     global WEBSOCKET_POLICY
 
+    config.register_service_factory('.services.auth_ticket_service_factory',
+                                    iface=pyramid_authsanity.interfaces.IAuthService)
```

Comment:
Oh, yes, so you do. Probably still nicer to use a dotted string in places you can so there isn't a perceived import dependency where one isn't needed.